### ISSUE TITLE
release steps

### ISSUE THREAD

**Issue by zassmin** [WomenWhoCode/meet-maynard#29]
## Problem

What do we need to do after https://github.com/WomenWhoCode/meet-maynard/pull/28 is complete?

## Solution
- migrate db
- schedule rake tasks daily in the following order
  1. `rake data_import:pro_group`
  1. `rake data_import:events`
  1. `rake data_import:rsvps`
- update tokens

## blocked by
https://github.com/WomenWhoCode/meet-maynard/issues/2

**Comment by isometimescode** [comment]
* Add log monitoring (papertrail?)
* Make sure Bugsnag is in production (I think its on staging...)
* Any automated deployment type tasks? Right now we aren't running the `rspec` tests automatically or anything. That's probably low priority if we aren't making a lot of rapid changes in this repo.
* Database backups should get enabled

**Comment by zassmin** [comment]
thanks @isometimescode 🚀 

created an issue, https://github.com/WomenWhoCode/meet-maynard/issues/30, for papertrail will cover the other points in initial release
